dll/win32/ws2_32/ws2_32.spec

```diff
@@ -115,3 +115,7 @@
 23  stdcall  socket(long long long)
 @ stdcall GetAddrInfoW(wstr wstr ptr ptr)
 @ stdcall GetNameInfoW(ptr long wstr long wstr long long)
+@ stdcall -version=0x600+ inet_ntop(long ptr str long)
+@ stdcall -version=0x600+ InetNtopW(long ptr wstr long)
```

Comment:
For these two functions, their 3rd parameter, although being a string, is a parameter used as _**output**_, i.e. on input the contents of this string buffer is undefined. Therefore, the 3rd-parameter should be `ptr` here instead of `str` or `wstr`.

To understand what I mean, you could test the following modification: in ws2_32 CMakeFiles.txt, change
`spec2def(ws2_32.dll ws2_32.spec ADD_IMPORTLIB)`
to
`spec2def(ws2_32.dll ws2_32.spec ADD_IMPORTLIB WITH_RELAY)`
and look at how the stubs in the auto-generated `ws2_32_stubs.c` file in the ws2_32's output build folder looks like.